script/deploy

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
```

Comment:
I gotcha. I'd write it in Python, but part of it is that Ruby has a very nice optparse library which could be used for our scripts.  I'll definitely keep maintaining this into the future.